src/main/java/com/revature/taskmaster/auth/AuthService.java

```diff
@@ -4,7 +4,9 @@
 import com.revature.taskmaster.common.exceptions.InvalidRequestException;
 import com.revature.taskmaster.users.UserDAO;
 import com.revature.taskmaster.users.UserResponse;
+import org.springframework.stereotype.Service;
 
+@Service
 public class AuthService {
 
     private final UserDAO userDAO;
```

Comment:
please include the @Autowired annotation for constructor injection - just to be explicit